addons/portal/__openerp__.py

```diff
@@ -22,12 +21,10 @@
 very handy when used in combination with the module 'share'.
     """,
     'data': [
-        'portal_data.xml',
-        'portal_view.xml',
+        'data/portal_data.xml',
+        'views/portal_views.xml',
         'wizard/portal_wizard_view.xml',
         'security/ir.model.access.csv',
     ],
-    'demo': ['portal_demo.xml'],
-    'auto_install': True,
```

Comment:
default value for auto_install is False, here required "auto_install": True............... ;)